libs/core/src/components/sage-link/sage-link.scss

```diff
@@ -0,0 +1,68 @@
+:host {
+  display: inline;
+
+  svg {
+    margin-left: 4px;
+  }
+}
+
+:host(:focus) {
+  border-radius: 6px;
+  outline: 4px solid #8ecafb;
+  outline-offset: 4px;
+}
+
+.sage-link {
+  align-items: center;
+  color: #202327;
+  display: inline-flex;
+  font-weight: inherit;
+}
+
+// We have a small consensus stating that
+// fonts will be REMs, box-model will be
+// discussed later px or (r)em's
+.sage-link--sm {
+  font-size: 12px;
+
+  svg {
+    height: 11px;
+    width: 11px;
+  }
+}
+
+.sage-link--md {
+  font-size: 14px;
+
+  svg {
+    height: 13px;
+    width: 13px;
+  }
+}
+
+.sage-link--lg {
+  font-size: 16px;
+
+  svg {
+    height: 15px;
+    width: 15px;
+  }
+}
+
+.sage-link--plain {
+  font-weight: 500;
```

Comment:
Same as above, I was under the impression that font weight should be derived from the parent element?